### PR TITLE
NMA-6142: Add new border colors to outlined text fields

### PR DESCRIPTION
**Pull request by sindrenm**
| Before |   After  |
|-|-|
| image | image |
| image | image |

### CODE REVIEW

File: sats-dna/src/main/kotlin/com/sats/dna/components/SatsOutlinedTextField.kt [hunk 1]
```diff
@@ -50,6 +56,57 @@ fun SatsOutlinedTextField(
...
+    fun colors(
+        textColor: Color = M2LocalContentColor.current.copy(M2LocalContentAlpha.current),
+        disabledTextColor: Color = textColor.copy(M2ContentAlpha.disabled),
+        backgroundColor: Color = Color.Transparent,
+        cursorColor: Color = M2MaterialTheme.colors.primary,
+        errorCursorColor: Color = M2MaterialTheme.colors.error,
+        focusedBorderColor: Color = SatsTheme.colors2.graphicalElements.border.focused,
+        unfocusedBorderColor: Color = SatsTheme.colors2.graphicalElements.border.default,
+        disabledBorderColor: Color = SatsTheme.colors2.graphicalElements.border.default,
+        errorBorderColor: Color = SatsTheme.colors2.graphicalElements.signal.error,
```
Comment (sindrenm) [inline sats-dna/src/main/kotlin/com/sats/dna/components/SatsOutlinedTextField.kt:70]: Our own defaults, can be overridden if needed. Should we allow overriding these?

File: sats-dna/src/main/kotlin/com/sats/dna/components/SatsOutlinedTextField.kt [hunk 2]
```diff
@@ -86,6 +143,101 @@ fun M3SatsOutlinedTextField(
...
+        unfocusedContainerColor: Color = Color.Unspecified,
+        disabledContainerColor: Color = Color.Unspecified,
+        errorContainerColor: Color = Color.Unspecified,
+        cursorColor: Color = Color.Unspecified,
+        errorCursorColor: Color = Color.Unspecified,
+        selectionColors: TextSelectionColors? = null,
+        focusedBorderColor: Color = SatsTheme.colors2.graphicalElements.border.focused,
+        unfocusedBorderColor: Color = SatsTheme.colors2.graphicalElements.border.default,
+        disabledBorderColor: Color = SatsTheme.colors2.graphicalElements.border.default,
+        errorBorderColor: Color = SatsTheme.colors2.graphicalElements.signal.error,
```
Comment (sindrenm) [inline sats-dna/src/main/kotlin/com/sats/dna/components/SatsOutlinedTextField.kt:163]: Our own defaults, can be overridden if needed. Should we allow overriding these?

File: sats-dna/src/main/kotlin/com/sats/dna/components/SatsOutlinedTextField.kt (hunk 1 above)
Comment (cecilieopperudsats) [reply sats-dna/src/main/kotlin/com/sats/dna/components/SatsOutlinedTextField.kt:70]: I guess overriding if needed should be ok, if it's needed it's needed 🪡

File: sats-dna/src/main/kotlin/com/sats/dna/components/SatsOutlinedTextField.kt (hunk 2 above)
Comment (cecilieopperudsats) [reply sats-dna/src/main/kotlin/com/sats/dna/components/SatsOutlinedTextField.kt:163]: Same as above

**Review by cecilieopperudsats — APPROVED** [review]
Yes! 🏑

**Pull request merged**